src/modules/70-pipeline/components/PipelineStudio/RightDrawer/RightDrawer.tsx

```diff
@@ -109,14 +101,14 @@ export const updateStepWithinStage = (
         if (parallelStep?.stepGroup?.identifier === processingNodeIdentifier) {
           parallelStep.stepGroup = processedNode as any
         } else if (parallelStep.step?.identifier === processingNodeIdentifier) {
-          parallelStep.step = processedNode
+          parallelStep.step = processedNode as any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232258)
```suggestion
          parallelStep.step = processedNode as unknown
```